src/main/java/co/cask/hydrator/salesforce/plugin/BaseSalesforceConfig.java

```diff
@@ -54,14 +58,18 @@
   @Macro
   private final String loginUrl;
 
+  @Description("Strategy to handle erroneous records")
```

Comment:
give example for what these should be set as, and what behaviors result from them?

Reply:
done